Tasks/DeployAzureResourceGroup/DtlServiceHelper.ps1

```diff
@@ -199,6 +204,7 @@ function Check-EnvironmentNameAvailability
     {
         Write-Verbose -Verbose "Checking machine group name availability"
 
+        import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
```

Comment:
Silimarly here. DevTestLabs dll is anyway imported in the main ps1 file, why import again.